copy-kibana-dashboard.rb

```diff
@@ -72,6 +76,7 @@ def parse_args
     ['--to-host', GetoptLong::OPTIONAL_ARGUMENT],
     ['--to-port', GetoptLong::OPTIONAL_ARGUMENT],
     ['--to-index', GetoptLong::OPTIONAL_ARGUMENT],
+    ['--to-saved-search-index', GetoptLong::REQUIRED_ARGUMENT],
```

Comment:
Or what does optional mean here? the whole option is optional or the argument to the option is optional? i can test tomorrow